argocon-demos/2023-how-to-train-an-llm-with-argo-workflows-and-hera/README.md

```diff
@@ -0,0 +1,19 @@
+# How to Train an LLM with Argo Workflows with Hera
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
+
+## The talk
+The talk recording [can be found here](https://youtu.be/nRYf3GkKpss).
+
+The slide deck for this talk can be found [here](assets/slide-deck.pdf).
+
+## The code
+The code for this talk is found under the [./example](example) folder.
```

Comment:
Should this link the repo?

Reply:
I cloned it into the examples folder happy to link to your repo as well